src/iter.rs

```diff
@@ -7,13 +7,24 @@ use heapless::String;
 pub struct MiniconfIter<M: ?Sized, const L: usize, const TS: usize> {
     marker: PhantomData<M>,
     state: [usize; L],
+    count: Option<usize>,
```

Comment:
Mind adding some docs for what these members are supposed to be? It took me a minute to grok what `count` is supposed to be. We should also document what `state` is

Reply:
Ack